### PR TITLE
Issue template: add references what to put in a bug report

### PR DESCRIPTION
**Pull request by fedepaol**
Expanding the troubleshooting guide so it's clearer to users what to try and how to collect the information required in a given issue.

### CODE REVIEW

**Comment by fedepaol** [comment]
Thanks to @Ornias1993, heavily inspired from https://github.com/truecharts/.github

**Comment by Ornias1993** [comment]
No problem, happy to be of inspiration! :)

File: .github/ISSUE_TEMPLATE/outdated_app.yaml
[the remark this answers is not in the document]
Comment (fedepaol) [reply .github/ISSUE_TEMPLATE/outdated_app.yaml:1]: nope just copy / paste

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      options:
+        - label: I've checked all open and closed issues and my issue is not there.
+          required: true
+        - label: This bug is reproducible when deploying MetalLB from the main branch
+          required: true
+        - label: I have read the [troubleshooting guide](https://metallb.universe.tf/troubleshooting/) and I am still not able to make it work
+          required: true
+        - label: I checked the logs and MetalLB is not discarding the configuration as not valid
+          required: true
+        - label: I enabled the debug logs, collected the information required from the cluster using the [collect script](https://raw.githubusercontent.com/metallb/metallb/main/troubleshooting/collect.sh) and attached will attach them to the issue
```
[the remark this answers is not in the document]
Comment (fedepaol) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:118]: I will use the future form because the user will have to attach them after filing the form

**Review by oribon — APPROVED** [review]
lgtm

**Pull request merged**